src/main/java/sirius/db/jdbc/StatementCompiler.java

```diff
@@ -176,6 +192,17 @@ private Object computeEffectiveParameterValue(String fullParameterName) throws S
 
         Object paramValue = context.get(parameterName);
         if (accessPath == null || paramValue == null) {
+            if (!context.containsKey(parameterName)) {
+                Databases.LOG.WARN("""
+                                           SQL query references an unknown paramter: %s! \
+                                           This is very likely an error and thus reported. \
+                                           Set unknown parameters to null to suppress this warning!
+                                           Query: %s
+                                           Location: %s""",
```

Comment:
Is the triple quotes + empty line intended?

Reply:
Thats the new java multiline text syntax

Reply:
yes they are - https://openjdk.java.net/jeps/355 for anyone else wondering

Reply:
https://www.baeldung.com/java-multiline-string

Reply:
Yep, this is a multiline string literal and yields a message like:
```
[2021-06-17T17:20:51.317] WARNING [jdbc - StatementCompiler.java:196] SQL query references an unknown paramter: filter! This is very likely an error and thus reported. Set unknown parameters to null to suppress this warning!
Query: SELECT * FROM test_a [WHERE a = ${filter}]
Location:
Execution Point in: main
---------------------------------------------------
sirius.kernel.async.ExecutionPoint.snapshot (ExecutionPoint.java:59)
sirius.kernel.async.ExecutionPoint.snapshot (ExecutionPoint.java:42)
sirius.db.jdbc.StatementCompiler.computeEffectiveParameterValue (StatementCompiler.java:205)
sirius.db.jdbc.StatementCompiler.compileSectionPart (StatementCompiler.java:167)
...
...
```